apps/forums/api/serializers.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework import serializers
+
+from apps.forums.models import Post, Thread
+from core.models.serializers import DrfModuleSerializer
+
+
+class ThreadDrfSerializer(serializers.ModelSerializer):
+    """Serializer for threads."""
+
+    class Meta:
+        model = Thread
+        fields = ['pk', 'updated_date']
+
+
+class PostDrfSerializer(serializers.ModelSerializer):
+    """Serializer for posts."""
+
+    class Meta:
+        model = Post
+        fields = [
+            'pk',
+            'content',
+            'title',
+            'date',
+            'author',
+            'parent_thread',
+        ]
+        read_only_fields = ['pk']
```

Comment:
> This might not be necessary for this PR, but jsyk we can use serializers as fields so that the frontend receives related objects, instead of just IDs. E.g. to include user data (name, avatar, etc) we can use [`UserSerializer`](https://github.com/ModularHistory/modularhistory/blob/MH-115/apps/users/api/serializers.py#L16) like this, I think:
> 
> ```python
> class PostDrfSerializer(serializers.ModelSerializer):
>     """Serializer for posts."""
> 
>     author = UserSerializer()
> 
>     class Meta:
>         ...
> ```

Thanks for letting me know! I'll keep that in mind and look into it for future tickets!